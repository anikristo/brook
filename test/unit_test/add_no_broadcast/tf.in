1 9
4 5 6 2 6 -3 -5 4 -22