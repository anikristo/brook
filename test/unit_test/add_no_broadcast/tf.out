1 9
4.30178 7.13193 7.94813 4.77421 9.36709 1.16784 -0.294989 9.87816 -15.2375
